### PR TITLE
Invert GitHub icon for bugs.python.org

### PR DESCRIPTION
**Pull request by xxyzz**
Invert GitHub icon inside the home page and issue page(login icon).

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -2362,6 +2362,14 @@ th {
 
 ================================
 
+bugs.python.org
+
+INVERT
+img[title="has PR"]
+img[title="GitHub"]
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:2369]: Should this rather be title="has patch"?

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:2369]: Yes... that should also be inverted(barely visible). This title="Github" doesn't seem to match any icons.

Comment (xxyzz) [reply src/config/dynamic-theme-fixes.config:2369]: title="Github" is at the left side of each [issue page](https://bugs.python.org/issue40255):
Screen Shot 2022-02-22 at 16 04 56

Invert the clip icon doesn't have any effect.

**Review by Gusted — APPROVED** [review]
LGTM

**Pull request merged**